.gitignore

```diff
@@ -12,4 +12,5 @@ node_modules
 .DS_Store
 .AppleDouble
 .LSOverride
-/lib
\ No newline at end of file
+/lib
+.env
```

Comment:
how about end of line ?

Reply:
IDE fails to add that for the dotfiles 👎

Reply:
:dancing_men: